internal/cmd/gocdk/internal/static/_assets/launchers/ecs.tf

```diff
@@ -1,17 +1,21 @@
+# This file creates an ECS cluster and its instances. The AWS Console creates
+# these resources for you automatically, but AWS does not expose an equivalent
+# API.
+
 # Create an ECR Docker repository.
 # The name used here will be part of the deployed Docker image name.
 resource "aws_ecr_repository" "default" {
-  name = "{{.ProjectName}}"
+  name = local.gocdk_random_name
 }
 
 # ECS Cluster
 
 resource "aws_ecs_cluster" "default" {
-  name = random_id.cluster_name.hex
+  name = local.gocdk_random_name
 }
 
 resource "random_id" "cluster_name" {
```

Comment:
You can delete this resource, it is no longer used.

Reply:
Done.